### PR TITLE
Timely v2 changes

### PR DESCRIPTION
**Pull request by sachinmjadhav**
* Remove close button
* Update packages

### CODE REVIEW

File: src/components/TimelyModal.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 
-import { ReactComponent as CloseIcon } from '../assets/close-icon.svg'
-
 export interface StylesDictionary {
   [Key: string]: React.CSSProperties
 }
 
 interface TimelyModalProps extends PropsWithChildren {
   isOpen: boolean
+  children: React.ReactNode
```
Comment (arifraveendran) [inline src/components/TimelyModal.tsx]: already extending propswithchildren right ?

**Review by arifraveendran — COMMENTED** [review]
rest looks good

File: example/src/App.tsx [hunk 2]
```diff
@@ -24,6 +24,22 @@ const App = () => {
       }
     })
   }
+
+  useEffect(() => {
+    if (typeof window === 'undefined') return
+
+    const listener = (event: any) => {
+      if (event.data?.from !== 'timely–link') return
+      window.location.href = event.data.action
```
Comment (arifraveendran) [inline example/src/App.tsx]: isn't it better to open these links in a new tab ?

**Review by arifraveendran — CHANGES_REQUESTED** [review]

File: example/src/App.tsx (hunk 2 above)
Comment (sachinmjadhav) [reply example/src/App.tsx]: Ok, should we not close the modal on link click?

Comment (arifraveendran) [reply example/src/App.tsx]: better not close it in case the user wants to check all the links.

**Review by arifraveendran — APPROVED** [review]

**Pull request merged**